components/brave_wallet_ui/components/desktop/views/nfts/nft-view.tsx

```diff
@@ -17,11 +20,17 @@ import { Nfts } from './components/nfts'
 
 interface Props {
   nftsList: BraveWallet.BlockchainToken[]
-  accounts: WalletAccountType[]
+  accounts: BraveWallet.AccountInfo[]
   onShowPortfolioSettings?: () => void
+  tokenBalancesRegistry: TokenBalancesRegistry | undefined
```

Comment:
maybe we should instead run the query to fetch this in this component?

Reply:
Since the `tokenBalancesRegistry` prop is forwarded from `PortfolioOverview`, I think there will be duplicate requests if we fetch here.

I agree it's cleaner though. We'll probably need to refactor the `tokenBalancesRegistry` in `PortfolioOverview` to only lookup fungible token balances in that case. Prefer doing such refactoring in another PR.